src/test/scala/uk/gov/nationalarchives/scannotifications/LambdaSpecUtils.scala

```diff
@@ -19,9 +19,11 @@ class LambdaSpecUtils extends AnyFlatSpec with Matchers with BeforeAndAfterAll w
     (critical, high, medium, low)
   }
 
-  val scanEvent1: ScanEvent = ScanEvent(ScanDetail("", ScanFindingCounts(Some(10), Some(100), Some(1000), Some(10000))))
-  val scanEvent2: ScanEvent = ScanEvent(ScanDetail("", ScanFindingCounts(Option.empty, Some(0), Option.empty, Some(10))))
-  val scanEvent3: ScanEvent = ScanEvent(ScanDetail("", ScanFindingCounts(Option.empty, Option.empty, Option.empty, Option.empty)))
+  val scanEvent1: ScanEvent = ScanEvent(ScanDetail("", List("latest"), ScanFindingCounts(Some(10), Some(100), Some(1000), Some(10000))))
+  val scanEvent2: ScanEvent = ScanEvent(ScanDetail("", List("latest"), ScanFindingCounts(Option.empty, Some(0), Option.empty, Some(10))))
+  val scanEvent3: ScanEvent = ScanEvent(ScanDetail("", List("latest"), ScanFindingCounts(Option.empty, Option.empty, Option.empty, Option.empty)))
+  val scanEvent4: ScanEvent = ScanEvent(ScanDetail("", List("anothertag"), ScanFindingCounts(Option.empty, Option.empty, Option.empty, Option.empty)))
+  val scanEvent5: ScanEvent = ScanEvent(ScanDetail("", List("latest"), ScanFindingCounts(Some(0), Some(0), Some(0), Some(0))))
```

Comment:
`scanEvent5` does not appear to be referenced anywhere. Is it being used?

Reply:
I've added it into the test table. I had to change the tag to make it different from one of the other tests.